doc/conf.py

```diff
@@ -292,6 +292,13 @@
 # This patterns also effect to html_static_path and html_extra_path
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store", "**.ipynb_checkpoints", "_templates*"]
 
+# Pages are not detected correct by ``make linkcheck``
+linkcheck_ignore = [
+    'https://data.kitware.com/#collection/55f17f758d777f6ddc7895b7/folder/5afd932e8d777f15ebe1b183',
+    'https://www.sciencedirect.com/science/article/abs/pii/S0309170812002564',
```

Comment:
So this middle one, the sciencedirect link is also weird.

During `make linkcheck` I get
```
(examples/00-load/terrain-mesh: line   28) -ignored- https://www.sciencedirect.com/science/article/abs/pii/S0309170812002564: 503 Server Error: Service Temporarily Unavailable for url: https://www.sciencedirect.com/science/article/abs/pii/S0309170812002564
```
In the browser it loads fine.
And with `requests.get` I get a 403. So pretty much the same as the researchgate link. Hardly surprising of course: Elsevier are jerks.

Which is all to say I agree we can't help but ignore these three links, unless we can eventually use a user agent for linkcheck that isn't rejected by sciencedirect and researchgate.

Reply:
For the record user agents don't seem to help either. (And even if they did, it would be quite questionable to try to mask that we are technically scraping these websites.)

Reply:
Thanks for the points and for the review!